src/api/app/components/sponsors_component.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class SponsorsComponent < ApplicationComponent
+  def initialize(config: CONFIG)
+    super
+    @config = config
+  end
+
+  def sponsors?
+    @config['sponsors'].present?
+  end
+
+  def sponsors
+    @config.fetch('sponsors', [])
```

Comment:
We can memoize this and use this method in `sponsors?`.

```suggestion
    @sponsors ||= @config.fetch('sponsors', [])
```